content/post/087-js-ipfs-0.42.md

```diff
@@ -0,0 +1,194 @@
+---
+date: 2020-04-14
+url: 2020-04-14-js-ipfs-0-42
+title: js-ipfs 0.42.0 released
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Smaller, faster, more reliable
+
+`js-IPFS@0.42.0` has been released, bringing a host of bug fixes and other small stability improvements.
+
+It reduces the size of the minified bundle by 7.5% for faster downloads and more responsive web apps!
```

Comment:
Can you add a highlight about monorepo work making it easier to contribute and code spelunk?

Reply:
I've added a bit at the start